<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myServer</class>
 <widget class="QMainWindow" name="myServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myServer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEditMasages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="transferType">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>broadcast</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>reciver</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="clientListCombo">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendFileBTN">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>send file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
